node/lib/util/open.js

```diff
@@ -83,12 +86,16 @@ exports.openOnCommit = co.wrap(function *(fetcher,
                                                                 metaRepo,
                                                                 submoduleName,
                                                                 submoduleUrl,
-                                                                templatePath);
+                                                                templatePath, 
+                                                                bare);
 
     // Turn off GC for the submodule
     const config = yield submoduleRepo.config();
     config.setInt64("gc.auto", 0);
 
+    if (bare) {
```

Comment:
Why are you even calling this function in this case?  Surely not in order to set gc.auto...

Reply:
call this function to init `subModuleAndRepo` if the repo is not even half-opened

Reply:
Oh, I was confused by the github ui, my bad.